front/lib/models.js

```diff
@@ -63,10 +63,16 @@ export const App = sequelize.define(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    isDeleted: {
```

Comment:
Thinking more about it, is it better to have an isDeleted or a visibility: deleted?

Pro of isDeleted: if you undelete you recover the visibility
Cons of isDeleted: new field that is a boolean (boolean have a tendency to become enums over time no?)